File: PRAKTIKUM PEMROGRAMAN I P1/MODUL 4/SOAL 3/PRAK403-2410817210002-Sebastian_Abe_Santoso.py
a = []
while len(a) != 2:
    b = input()
    a.extend(b.split())
init1, init2 = a[:2]
init1 = int(init1); init2 = int(init2)
no1 = init1; no2 = init2
step = -1 if init1 > init2 else 1
while no1 != init2 and no2 != init1:
    print(no1, no2,"-",end =' ')
    no1 += step; 
    no2 -= step; 
print(no1, no2)

File: MODUL 4/SOAL 3/PRAK403-2410817210002-Sebastian_Abe_Santoso.c
#include <stdio.h>
int main() {
    int init1, init2;
    scanf("%d %d", &init1, &init2);
    int no1 = init1, no2 = init2;
    int step = (init1 > init2) ? -1 : 1; 
    while (no1 != init2 && no2 != init1) {
        printf("%d %d - ", no1, no2);
        no1 += step; 
        no2 -= step; 
    } printf("%d %d\n", no1, no2);
    return 0;
}